### ISSUE TITLE
Don't fail elements in aria-busy=true

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#923]
As discussed in #874. Content wrapped in `aria-busy="true"` shouldn't be failing certain rules. The big question is, which rules do we allow "needs review" instead of "fail" when wrapped inside aria-busy? It seems like this could be applicable to most rules.

One way we could do this is to do something like `rule.excludeHidden`. If `"ignoreAriaBusy"` isn't set to `true`, results from that rule should become "incomplete" instead of "fail".

**Comment by dylanb** [comment]
I don't think the result should be incomplete, it should be a pass

**Comment by marcysutton** [comment]
What about listboxes with dynamic children that haven't yet been appended, like in #383? We were worried users would just slap `aria-busy="true"` to get the test to pass without actually testing the options.

**Comment by WilcoFiers** [comment]
That's exactly why I'm proposing we put those into incomplete.

**Comment by WilcoFiers** [comment]
I have not seen this come up in a long time. I'm closing the issue, doesn't seem anyone's using aria-busy this way.

**Comment by muan** [comment]
👋🏻 I subscribe to this issue because we'd love to use `aria-busy`. In GitHub UI there are many deferred loading content and `aria-required-children` fails before the content is loaded in. We current have workaround that conditionally skip scanning when container shows up, and bind Axe.run on child content load instead. We haven't prioritize adding `aria-busy` because it doesn't seem to be well-supported. `axe-core` understanding it certainly will bump this up for us, and maybe then the added usage can help signal to browsers to expose the state.
